Description: Fix compile time warning about 'use of assignment suppression and length modifier together'
Author: James Troup <james.troup@canonical.com>
--- a/nicstat.c
+++ b/nicstat.c
@@ -1476,10 +1476,10 @@ load_netstat(FILE *netstat)
 		    "DelayedACKLost ListenOverflows ListenDrops ",
 		    p, 273) == 0) {
 			/* We are after field 20 and 21 */
-			int n = fscanf(netstat, "TcpExt: %*lld %*lld %*lld "
-				"%*lld %*lld %*lld %*lld %*lld %*lld %*lld "
-				"%*lld %*lld %*lld %*lld %*lld %*lld %*lld "
-				"%*lld %*lld %lld %lld ",
+			int n = fscanf(netstat, "TcpExt: %*d %*d %*d "
+				"%*d %*d %*d %*d %*d %*d %*d "
+				"%*d %*d %*d %*d %*d %*d %*d "
+				"%*d %*d %lld %lld ",
 				&ll[0], &ll[1]);
 			if (n == 2)
 				g_tcp_new->listenDrop = ll[0] + ll[1];
